metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -1155,6 +1155,13 @@ def _get_view_definition(self, inspector: Inspector, schema: str, view: str) ->
         except NotImplementedError:
             return ""
 
+    def get_view_default_db_schema(
+        self, dataset_name: str, _inspector: Inspector, _schema: str, _view: str
+    ) -> Tuple[Optional[str], Optional[str]]:
+        # Some databases use different implicit databases/schemas for unqualified
```

Comment:
"some databases"?
this method consistently returns None, None, how that applies to some databases and which are those databases?